src/plugins/objects/realtimeobject.ts

```diff
@@ -80,11 +80,24 @@ export class RealtimeObject {
    * This is useful when working with multiple channels with different underlying data structure.
    */
   async get<T extends Record<string, API.Value>>(): Promise<API.PathObject<API.LiveMap<T>>> {
-    this.throwIfInvalidAccessApiConfiguration(); // RTO1a, RTO1b
+    this._throwIfMissingChannelMode('object_subscribe');
+
+    let syncedPromise;
+    // grab the synced promise now before attaching to avoid race condition
+    if (this._state !== ObjectsState.synced) {
+      syncedPromise = this._eventEmitterInternal.once(ObjectsEvent.synced);
+    }
+
+    // implicit attach before proceeding
+    await this._waitAttached();
```

Comment:
Presence has additional handling for `SUSPENDED`; do we not want that?

Reply:
Ah fair enough. From what I can see the DR doesn't explain why we chose to deviate from presence for `SUSPENDED`; do you remember why we did?